src/components/FeaturedEvent.tsx

```diff
@@ -2,9 +2,9 @@ import { FC } from "react";
 import FeaturedComponentBase from "./FeaturedComponentBase";
 import { Event } from "../model";
 import { formatDate } from "../utils/date";
-import { browserParse, transformToPortableText } from "@kontent-ai/rich-text-resolver";
+import { transformToPortableText } from "@kontent-ai/rich-text-resolver";
 import { PortableText } from "@portabletext/react";
```

Comment:
as mentioned previously:

`import { PortableText } from "@kontent-ai/rich-text-resolver/utils/react";`

will provide default resolution for `sup` and `sub`, so you can get rid of it in richtext.ts